MANIFEST.in

```diff
@@ -2,3 +2,6 @@ recursive-include thinc *.cu *.pyx *.pxd
 include LICENSE
 include README.md
 prune tmp/
+include thinc/tests/mypy/configs/*.ini
+include thinc/tests/mypy/outputs/*.txt
+include thinc/py.typed
```

Comment:
This empty file `py.typed` included in the package is what makes it PEP 561 compatible and tells [mypy it can actually use the types included to check code written by others](https://mypy.readthedocs.io/en/latest/installed_packages.html#making-pep-561-compatible-packages).